diag_manager/diag_output.F90

```diff
@@ -315,6 +316,8 @@ SUBROUTINE write_axis_meta_data(file_unit, axes, fileob, time_ops, time_axis_reg
     integer                                    :: id_axis_current
     logical :: is_time_axis_registered
     integer :: istart, iend
+    integer :: gstart, dgnd, cstart, cend !< Start and end of global and compute domains
```

Comment:
You define dgnd here, but it doesn't seem to get used.

Reply:
`dgnd` should be `gend`

Reply:
... which would cause an issue because there already is a `gend`